### ISSUE TITLE
[Bug?]: @redwoodjs/internal depends on typescript and prettier

### ISSUE THREAD

**Issue by rmzg-ll** [redwoodjs/redwood#7265]
### What's not working?

Installing this package by itself pulls in both prettier and typescript which combine for over 100megs of dependencies, it'd be nice if they were either moved to devDependencies or removed entirely.

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by KhuloodHassan** [comment]
Is this issue open?

**Comment by jtoar** [comment]
@KhuloodHassan yes it's still open. `@redwoodjs/internal` still depends on both these packages, and while it still needs their functionality for certain things, most of that functionality is dev-time, and not needed for build and prod in containerized environments. Happy to follow up with more details if you're interested; but if you were only wondering if it was a problem or not I may have already given you too many

**Comment by KhuloodHassan** [comment]
@jtoar Was asking so I can contribute! Can I get more detail? I'm currently learning React and Typescript so I might need some help but would to contribute!
